aes/aes-soft/src/fixslice32.rs

```diff
@@ -62,12 +62,21 @@ pub(crate) fn aes128_key_schedule(key: &GenericArray<u8, U16>) -> FixsliceKeys12
     }
 
     // Adjust to match fixslicing format
-    for i in (8..72).step_by(32) {
-        inv_shift_rows_1(&mut rkeys[i..(i + 8)]);
-        inv_shift_rows_2(&mut rkeys[(i + 8)..(i + 16)]);
-        inv_shift_rows_3(&mut rkeys[(i + 16)..(i + 24)]);
+    #[cfg(feature = "semi_fixslice")]
```

Comment:
It doesn't look like you have a `semi_fixslice` feature declared in Cargo.toml?

I'm also not sure how I feel about a "feature" making things slower.

Reply:
Sorry, wasn't aware that was a thing. I can add it. As to making things slower, there is the already-existing example of the "no_unroll" feature (in the kuznyechik implementation) that makes the code smaller and slower.

Is it possible in rust to have a public feature like "small_code" that is translated internally in each algorithm to whatever local features make the code smaller?

Reply:
I wish there were better way to hint at these sort of things in Rust, but unfortunately there aren't AFAIK.